synapse/federation/transport/server.py

```diff
@@ -1100,7 +1100,7 @@ class FederationGroupsCategoryServlet(BaseFederationServlet):
     """Add/remove/get a category in a group
     """
     PATH = (
-        "/groups/(?P<group_id>[^/]*)/categories/(?P<category_id>[^/]+)$"
+        "/groups/(?P<group_id>[^/]*)/categories/(?P<category_id>[^/]+)/*$"
```

Comment:
this doesn't look right. surely we should not be accepting trailing slashes where we do not currently have them?

Reply:
What do we want exactly? And wasn't the goal to be permissive about trailing slashes everywhere?